text/0124-documents-api.md

```diff
@@ -451,4 +453,5 @@ The auth layer can return the following errors if Meilisearch is secured (a mast
 N/A
 
 ## 5. Future Possibilities
-N/A
\ No newline at end of file
+
+- Introduce a way to reject fields from a document in the response. e.g. `?fields=-createdAt`
```

Comment:
This option is really interesting, but I think it could add a lot of complexity,
The jsonapi spec specifies this behavior as sparse fieldsets: https://jsonapi.org/format/#fetching-sparse-fieldsets but I did not find a specific part about having the negation behavior of this option, so I think if they don't have it we don't need it too :)

Reply:
Yes, this is a bonus that may not be useful. I see it in case you have a lot of fields, and instead of specifying the ones you want, it becomes easier to remove the ones you don't want.